tests-beta/android/MASVS-PRIVACY/MASTG-TEST-02te.md

```diff
@@ -0,0 +1,50 @@
+---
+platform: android
+title: App Exposing Access and Verification Codes in Text Input Fields
```

Comment:
We should generalize I think, this isn't only about "Access and Verification Codes" but also passwords, PINs, etc., right?

Please fix the content below as well.

```suggestion
title: App Exposing User Authentication Data in Text Input Fields
```